modules/web-console-cso.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * web_console/scan-from-web-console.adoc
+
+[id="web-console-cso_{context}"]
+= Run Container Security Operator in {product-title}
+
+To start the Container Security Operator, go to the {product-title}
```

Comment:
Consider condensing this some, like "To start the Container Security Operator, install it from OperatorHub in the {product-title} web console."

I don't LOVE my suggestion, but I'm not a fan of the "go to the console" part

Reply:
I did this: "You can start the Container Security Operator from the {product-title}
web console by selecting and installing that Operator from the Operator Hub,
as described here."